### PR TITLE
Update test.md

### PR DESCRIPTION
**Pull request by tedhudek**

### CODE REVIEW

**Review by tedhudek — COMMENTED** [review]
This is a nice article.

File: test.md [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
-To comment on an older line, copy a link to that line and then paste the link into a commit comment.
-
-Or use blame to find the commit where the line was added, and add the comment to that commit.
+* You can do per-line comments that are associated with a commit, or with a pull request.
+* To comment on an older line, copy a link to that line and then paste the link into a commit comment.
+* Or use blame to find the commit where the line was added, and add the comment to that commit.
 
 If commenting on a PR, the reviewer can approve or request changes from the PR.  After the submitter makes changes, GitHub shows the reviewer only changes since the most recent review.
 
-Let's add another line.
```
Comment (tedhudek) [inline test.md:11]: In https://github.com/tedhudek/sandbox/commit/9504e922ec724ea4d850693d03231398b5fd2bff#diff-b128b7694e4de962e0088bcb1bcb254fR14 there was content here.

Comment (tedhudek) [inline test.md]: This is another conversation.

Comment (tedhudek) [reply test.md:11]: This is a reply to an existing comment.

File: test.md [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
 
 This file is for testing per-line commenting.
 
-You can do per-line comments that are associated with a commit, or with a pull request.
-
-To comment on an older line, copy a link to that line and then paste the link into a commit comment.
-
-Or use blame to find the commit where the line was added, and add the comment to that commit.
+* You can do per-line comments that are associated with a commit, or with a pull request.
+* To comment on an older line, copy a link to that line and then paste the link into a commit comment.
```
Comment (dondi) [inline test.md:6]: I don’t know if you were aware of this, but when GitHub is displaying code with line numbers, you can click on the line number and the URL becomes a direct link to that line, like this:

https://github.com/dondi/bazaar/blob/master/boxes-with-physics/boxes-with-physics.js#L21

(nothing in particular there, just a random file I picked out of one of my public repos)

You can also shift-click across a range of lines for a multi-line link.

**Review by dondi — COMMENTED** [review]
Great summary! Just for demonstration purposes I will leave this as a “Comment” review. In a more formal process, the reviewer can say “Approve” or “Request changes.”

File: test.md (hunk 2 above)
Comment (tedhudek) [reply test.md:6]: Very cool indeed.  Thanks @dondi for all the tips

**Pull request closed without merging**